lib/modules/hosts/vidlit.js

```diff
@@ -7,15 +7,15 @@ export default {
 	logo: 'https://vidl.it/favicon.ico',
 	detect: ({ href }) => (/^https?:\/\/vidl.it\/([A-Za-z0-9]+)$/i).exec(href),
 	async handleLink(href, [, hash]) {
-		const meta = await ajax({
-			url: `https://vidl.it/embed/meta/${hash}`,
+		const oEmbed = await ajax({
+			url: `https://vidl.it/oembed?url=${encodeURIComponent(href)}&maxwidth=640&format=json`,
```

Comment:
Small improvement: if you change this to 

```js
url: 'https://vidl.it/oembed',
data: { url: href, maxwidth: 640, format: 'json' },
```

it'll build the query string (with `encodeURIComponent`) for you.